src/bionics.cpp

```diff
@@ -2113,14 +2108,25 @@ bool Character::can_install_bionics( const itype &type, player &installer, bool
     } else {
         if( !g->u.query_yn(
                 _( "WARNING: %i percent chance of failure that may result in damage, pain, or a faulty installation!  Continue anyway?" ),
-                ( 100 - static_cast<int>( chance_of_success ) ) ) ) {
+                ( 100 - chance_of_success ) ) ) {
             return false;
         }
     }
 
     return true;
 }
 
+float Character::env_surgery_bonus( int radius )
+{
+    float bonus = 1.0;
+    for( const tripoint &cell : g->m.points_in_radius( pos(), radius ) ) {
+        if( g->m.furn( cell )->surgery_skill_multiplier ) {
+            bonus *= *g->m.furn( cell )->surgery_skill_multiplier;
+        }
+    }
+    return bonus;
```

Comment:
A perfect surgeon suddenly becomes a mediocre one if there is a bad one nearby?

Reply:
It depends what the multiplier represents and how it is used. Like if you have a wooden counter top as the autodoc couch with a 0.5 multiplier and a life monitor next to the player with a 1.2 multiplier it would make sense to get an effect from both.

Reply:
Yes but that needs to be explicit, because being surrounded by obsolete autodocs and one recent one should have no effect.